coffee/utils.py

```diff
@@ -624,6 +592,184 @@ def to_for(self, itspaces, dims=None, stmts=None):
         return loops
 
 
+###############################################################
+# Utilities for tracking the global impact of transformations #
+###############################################################
+
+
+class StmtTracker(OrderedDict):
```

Comment:
Ah right I had forgotten about it. It's because I need access by value too at some point